File: lab2/cpp/Task3.cpp
#include <iostream>

using namespace std;

int main() {
  int n;

  cout << "Enter a number between 3 and 9: ";
  cin >> n;

  while (n < 3 || n > 9) {
    cout << "Invalid input. Please enter a number between 3 and 9: ";
    cin >> n;
  }

  for (int i = 1; i <= 2 * n - 1; i++) {
    int k = n - abs(n - i);
    cout << string(k, '*') << "\n";
  }
  return 0;
}

File: lab2/py/task3.py
n = int(input("Please enter a number between 3 and 9: "))

while n < 3 or n > 9:
    n = int(input("Invalid Value.Please enter a number between 3 and 9: "))

for i in range(1, 2 * n):
    k = n - abs(n - i)
    print('*' * k)
